scripts/setup/macos-10.15/install_deps.sh

```diff
@@ -10,7 +10,7 @@ set -eux
 #brew update
 
 # Install dependencies via `brew`
-brew install universal-ctags wget
+brew install universal-ctags wget jq
```

Comment:
Why is jq required? I think you used it for the script that gathers data, but you didn't commit that script

Reply:
Oops. Thanks for catching it. I will add the statistics gathering command

Reply:
If we're clean now, I'm not sure we need it. But at any rate we don't need to add jq to CI if it's not being used

Reply:
We are not clean. This PR papers over the actual warning by turning off the lints. 

I believe this data should be tracked, but I agree you don't need to install jq on everything. Clippy check is only running for Ubuntu-20.04.

Reply:
Addressed with the following:
stats print script: 37cd8030b4492a07630401e74e0ffdf70eb21abd
remove jq install from OSX: 7803971d1f646310f146c3658f284a552feb9af5

As of now, no stats will print b/c lints are turned off: 

P.S. Wait what, did GH just suggest a fix??? 
```suggestion
brew install universal-ctags wget jq
```